src/realm/dictionary.cpp

```diff
@@ -44,6 +47,11 @@ void validate_key_value(const Mixed& key)
 
 // Dummy cluster to be used if dictionary has no cluster created and an iterator is requested
 static DictionaryClusterTree dummy_cluster(nullptr, type_Int, Allocator::get_default(), 0);
+static std::mt19937_64 generator(std::chrono::system_clock::now().time_since_epoch().count());
```

Comment:
What's wrong with std::default_random_engine? And should this be seeded from std::random_device rather than the system clock? Also, as I ask below, do we need this? Could this be an atomic int64_t?

Reply:
Randomness dropped.